content/en/docs/tasks/administer-cluster/kms-provider.md

```diff
@@ -78,7 +120,20 @@ Then use the functions and data structures in the stub file to develop the serve
 
 * protocol: UNIX domain socket (`unix`)
 
-  The gRPC server should listen at UNIX domain socket.
+  The plugin is implemented as a gRPC server that listens at UNIX domain socket. The plugin deployment should create a file on the file system to run the gRPC unix domain socket connection. The API server (gRPC client) is configured with the KMS provider (gRPC server) unix domain socket endpoint in order to communicate with it. 
+
+##### KMS v2 {#developing-a-kms-plugin-gRPC-server-notes-kms-v2}
+* kms plugin version: `v2alpha1`
+
+  In response to procedure call Status, a compatible KMS plugin should return `v2alpha1` as `StatusResponse.Version`, "ok" as `StatusResponse.Healthz` and a keyID (KMS KEK ID) as `StatusResponse.KeyID`
+
+* message version: `v2alpha1`
+
+  All messages from KMS provider have the version field set to current version v2alpha1.
```

Comment:
We no longer have the version field in `EncryptRequest` or `DecryptRequest` for KMS v2, so this can be removed.